### PR TITLE
feat(*): upgrade to the latest emulator updates

### PR DESCRIPTION
**Pull request by dackers86**
fixes: https://github.com/firebase/extensions/issues/689

### CODE REVIEW

File: firestore-send-email/functions/package.json [hunk 1]
```diff
@@ -17,15 +20,16 @@
   "dependencies": {
     "@types/node": "^12.6.9",
     "@types/nodemailer": "^6.2.1",
-    "firebase-admin": "^9.4.1",
-    "firebase-functions": "^3.13.2",
+    "firebase-admin": "^9.3.0",
```
Comment (huangjeff5) [inline firestore-send-email/functions/package.json:23]: We decreased the version on firebase-admin? Why?

**Comment by huangjeff5** [comment]
LGTM, please wait for Joe's review too.

**Review by huangjeff5 — APPROVED** [review]

File: firestore-send-email/functions/package.json [hunk 2]
```diff
@@ -8,7 +8,10 @@
     "build:watch": "npm run clean && tsc --watch",
     "clean": "rimraf lib",
     "compile": "tsc",
-    "test": "jest",
+    "local:emulator": "cd ../../_emulator && firebase emulators:start -P demo-test",
+    "test": "cd ../../_emulator && firebase emulators:exec jest -P demo-test",
+    "test:local": "concurrently \"npm run local:emulator\" \"jest\"",
```
Comment (joehan) [inline firestore-send-email/functions/package.json:13]: Is there any potential for a race condition here? ie can the tests start running before the emulator starts up?

**Review by joehan — APPROVED** [review]
LGTM with one question

File: firestore-send-email/functions/package.json (hunk 2 above)
Comment (dackers86) [reply firestore-send-email/functions/package.json:13]: Yes, there could be. I'm currently refining in the background, we could for example run a `sh` script, something like....

```bash
#!/bin/bash
if ! [ -x "$(command -v firebase)" ]; then
  echo "❌ Firebase tools CLI is missing."
  exit 1
fi

until curl --output /dev/null --silent --fail http://localhost:9099; do
  echo "Waiting for Firestore emulator to come online..."
  sleep 2
done

echo "Firestore emulator is online!" 

cd ../functions
jest --watch

```

**Pull request merged**